simtools/layout/geo_coordinates.py

```diff
@@ -0,0 +1,163 @@
+import logging
+
+import numpy as np
+import pyproj
+
+
+class GeoCoordinates:
+    """
+    Geospatial Coordinate systems.
+    Defines UTM, WGS84 and local (CORSIKA) coordinate systems.
+
+    """
+
+    def __init__(self):
+        """
+        Initialize GeoCoordinates
+
+        """
+
+        self._logger = logging.getLogger(__name__)
+
+    def crs_utm(self, epsg):
+        """
+        UTM coordinate system definition.
+
+        Parameters
+        ----------
+        epsg: int
+            EPSG code for UTM zone.
+
+        Returns
+        -------
+        pyproj.CRS
+            UTM coordinate system.
+
+        """
+        if epsg:
+            crs_utm = pyproj.CRS.from_user_input(epsg)
+            self._logger.debug(f"UTM coordinate system: {crs_utm}")
+            return crs_utm
+
+        return None
+
+    @staticmethod
+    def crs_wgs84():
+        """
+        WGS84 coordinate system definition.
+
+        Returns
+        -------
+        pyproj.CRS
+            WGS84 coordinate system.
+
+        """
+        return pyproj.CRS("EPSG:4326")
```

Comment:
Not my area of expertise so I can only ask, this hardcoded EPSG is OK? Applicable for both sites?

Reply:
Yes, EPSG:4326 is the lat/long coordinate system and the same everywhere on the globe.